vm/map.c

```diff
@@ -1193,12 +1194,12 @@ int _map_init(vm_map_t *kmap, vm_object_t *kernel, void **bss, void **top)
 
 	(*bss) += poolsz;
 	lib_printf("vm: Initializing memory mapper: (%d*%d) %d\n", map_common.nfree, sizeof(map_entry_t), poolsz);
-
-
+#ifdef NOMMU
```

Comment:
We don't need this `ifdef` if `_map_mapsInit` returns zero for MMU targets

Reply:
Done